tests/util.go

```diff
@@ -0,0 +1,56 @@
+package tests
+
+import (
+	"fmt"
+	"strings"
+	"time"
+	"io/ioutil"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	ktests "kubevirt.io/kubevirt/tests"
+)
+
+var KubeVirtOcPath = ""
+
+func ProcessTemplateWithParameters(srcFilePath, dstFilePath string, params ...string) string {
+	By(fmt.Sprintf("Overriding template from %s to %s", srcFilePath, dstFilePath))
+	args := []string{"process", "-f", srcFilePath}
+	args = append(args, params...)
+	out, err := ktests.RunOcCommand(args...)
+	Expect(err).ToNot(HaveOccurred())
+	filePath, err := writeJson(dstFilePath, out)
+	Expect(err).ToNot(HaveOccurred())
+	return filePath
+}
+
+func CreateResourceWithFilePath(resourceName, resourceType, filePath string) {
+	By(fmt.Sprintf("Creating %s:%s from JSON file via oc-create command", resourceType, resourceName))
+	_, err := ktests.RunOcCommand("create", "-f", filePath)
+	Expect(err).ToNot(HaveOccurred())
+	Eventually(func() bool {
+		out, err := ktests.RunOcCommand("get", resourceType)
+		Expect(err).ToNot(HaveOccurred())
+		return strings.Contains(out, resourceName)
+	}, time.Duration(2)*time.Minute).Should(BeTrue(), "Timed out waiting for ovm to appear")
```

Comment:
wouldn't this err msg depend on the resource you are waiting for?

Reply:
done